### DISCUSSION TITLE
Не приходит письмо подтверждения при регистрации на ovk.to

### DISCUSSION THREAD

**Discussion by pavlikmac** [OpenVK/openvk#1101] [Q&A]
На корпоративную почту не приходит письмо подвтерждения. Почтовый сервер сообщает что: warning: hostname mail.openvk.uk does not resolve to address 92.38.166.201, а затем происходит ошибка: SSL_accept error from unknown[92.38.166.201]: -1
dig говорит, что mail.openvk.uk запаркован (прописан CNAME к pixie.porkbun.com), но PTR у IP 92.38.166.201 указывает на mail.openvk.uk, хотя: mail.ovk.to IN A 92.38.166.201. Смените пожалуйста PTR у ip, чтобы он указывал на mail.ovk.to

**Comment by WerySkok** [comment]
Мне кажется, что требовать настройки PTR на сервере как-то кринжовато. Но тем не менее, сейчас обратимся к провайдеру, чтоб поменял.

**Reply by pavlikmac** [reply]
Я на своей стороне испробовал все, что можно и нельзя, но к сожалению мой сервер отказался принять почту от Вашего

**Reply by pavlikmac** [reply]
Другой причины отказа по STARTTLS я не вижу

**Comment by celestora** [comment]
Обновили PTR запись.

**Reply by pavlikmac** [reply]
Спасибо. Буду пробовать.
